tests/gnrc_netif_delayed_start/Makefile

```diff
@@ -0,0 +1,25 @@
+include ../Makefile.tests_common
+
+# Modules to include:
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+USEMODULE += at86rf215
```

Comment:
Why is this needed? Shouldn't it be `USEMODULE += netdev_default`

Reply:
Ops sorry, I added that when I tested on `same54-xpro` and forgot to remove it again.

Reply:
Still needs `USEMODULE += netdev_default`